### ISSUE TITLE
W: No zstd in /usr/bin:/sbin:/bin, using gzip

### ISSUE THREAD

**Issue by adrelanos** [grml/grml-debootstrap#211]
During initramfs creation.

> W: No zstd in /usr/bin:/sbin:/bin, using gzip

Should `zstd` get installed by default?

**Comment by mika** [comment]
Yeah, I think we can start shipping `zstd` by default. But given that it started to show up in 2015 within Debian, it would be interesting to know since which release it's available. (I'm not strictly saying we shouldn't modify the main `packages`, but I'm also reluctant about breaking backwards comparability. But maybe it's also good time to think about ways how to implement a better `packages` file handling, related to e.g. https://github.com/grml/grml-debootstrap/issues/44 + https://github.com/grml/grml-debootstrap/issues/98).

**Comment by paulmenzel** [comment]
It’s in Debian 10 (buster). Are older releases supported by grml-debootstrap?

**Comment by mika** [comment]
> It’s in Debian 10 (buster). Are older releases supported by grml-debootstrap?

Yes, see `releasetable.txt`:

```
.Current status
[width="40%",frame="topbot",options="header"]
|======================
|Release  |Status
|jessie   |works[1]
|stretch  |works[1]
|buster   |works[1]
|bullseye |works
|bookworm |works
|sid      |works[2]
|======================
```

(Reworking the package handling to make this per Debian release is on our wish list)

**Comment by paulmenzel** [comment]
From [Debian BTS bug report #882244 (*zstd: add zstd package to jessie-backports*)](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=882244), I’d say it is *not* present in Debian 8 (jessie) proper and only in `jessie-backports`.

**Comment by paulmenzel** [comment]
[ITP](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=806767)
